fiorella/src/layout/theme.liquid

```diff
@@ -32,7 +32,7 @@
 
   {% include 'social-meta-tags' %}
 
-  {{ 'theme.scss.css' | asset_url | stylesheet_tag }}
+  {{ 'main.scss.css' | asset_url | stylesheet_tag }}
```

Comment:
Porque importas css aqui? Eso es desde theme.scss no?

Reply:
implementamos la arquitectura "The 7-1 Pattern" que viene en la documentación de sass.
https://sass-guidelin.es/#architecture